OpenIDPlugin.php

```diff
@@ -0,0 +1,332 @@
+<?php
+
+namespace APP\plugins\generic\openid;
+
+use APP\core\Application;
+use APP\facades\Repo;
+use APP\plugins\generic\openid\forms\OpenIDPluginSettingsForm;
+use APP\plugins\generic\openid\handler\OpenIDHandler;
+use APP\plugins\generic\openid\handler\OpenIDLoginHandler;
+use PKP\core\PKPApplication;
+use PKP\linkAction\LinkAction;
+use PKP\linkAction\request\AjaxModal;
+use PKP\plugins\GenericPlugin;
+use PKP\plugins\Hook;
+use PKP\core\JSONMessage;
+use APP\template\TemplateManager;
+
+require_once(__DIR__ . '/vendor/autoload.php');
+
+/**
+ * This file is part of OpenID Authentication Plugin (https://github.com/leibniz-psychology/pkp-openid).
```

Comment:
I'd suggest homogenizing the code headers with ours -- i.e. refer to the license by name, and add a `LICENSE` document with the body of the license (this is more machine-readable). Credit Leibniz Psychology in the code header and preserve their copyright. That'll also result in this (wrong) URL being removed.

Reply:
Changed it to something like this 

```
/**
 * @file OpenIDPlugin.php
 *
 * Copyright (c) 2020 Leibniz Institute for Psychology Information (https://leibniz-psychology.org/)
 * Copyright (c) 2023 Simon Fraser University
 * Copyright (c) 2023 John Willinsky
 * Distributed under the GNU GPL v3. For full terms see the file LICENSE.
 *
 * @class OpenIDPlugin
 *
 * @ingroup plugins_generic_openid
 *
 * @brief OpenIDPlugin class for plugin and handler registration
 */
```

and added the LICENSE file in the root of the plugin's folder.